test.py

```diff
@@ -278,6 +282,22 @@ def esrgan(imgs, model_name):
             output1 = process(img1)
             output2 = process(img2)
             alpha = 1 - np.mean(output2-output1, axis=2)
+
+            if args.binary_alpha == 'True':
```

Comment:
Can this just be `if args.binary_alpha:`?

Reply:
Strings are coerced to boolean. If a user supplies say, binary_alpha False, and we don't expicitly check, it'll evaluate to true and execute the code block.

Reply:
Couldn't you make it a flag where you can just do `--binary_alpha` like what I did with the seamless flag?